AWS/aws-cloudfront/_meta/smart-descriptions.json

```
@@ -0,0 +1,151 @@
+[
+  {
+    "value": "{source.ip} called domain {destination.domain} and get the response from the nearest edge cache in {server.geo.name}",
+    "conditions": [
+      {
+        "field": "event.action",
+        "value": "Hit"
+      },
+      {
+        "field": "source.ip"
+      },
+      {
+        "field": "destination.domain"
+      },
+      {
+        "field": "server.geo.name"
+      }
+    ]
+  },
```

I suggest simplifying the description:
```suggestion
  {
    "value": "Delivered resource {url.path} to {source.ip} from the edge server {server.geo.name}",
    "conditions": [
      {
        "field": "event.action",
        "value": "Hit"
      },
      {
        "field": "source.ip"
      },
      {
        "field": "url.path"
      },
      {
        "field": "server.geo.name"
      }
    ]
  },